core/shared/src/main/scala/org/http4s/Message.scala

```diff
@@ -117,6 +117,11 @@ sealed trait Message[F[_]] extends Media[F] { self =>
   def withEmptyBody: Self =
     withBodyStream(EmptyBody).transformHeaders(_.removePayloadHeaders)
 
+  /**
+    */
```

Comment:
Probably something was intended to add here?

Reply:
Maybe worth mentioning it doesn't update any existing `Content-Length` header.  It's kind of a dangerous function, but as seen elsewhere in this PR, it is useful!

Reply:
So, it does not update any existing `Content-Length` header, just as the existing public method `withBodyStream` does not, so I was using that precedent.

Nevertheless, and since this is intended to be an Http4S utility, I have marked it as private to http4s.